packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/tooltip/tooltip.css

```diff
@@ -117,3 +117,21 @@
 		}
 	}
 }
+
+.ck[dir="rtl"] .ck.ck-tooltip {
```

Comment:
This should not be handled in CSS like this. Instead, this should be done [at the JS level](https://github.com/ckeditor/ckeditor5/blob/i/8335/packages/ckeditor5-html-embed/src/htmlembedediting.js#L350) based on `locale.uiLanguageDirection` (`locale.uiLanguageDirection === 'rtl' ? 'se' : 'sw'`). To do that we need a new `.ck-tooltip_se` style [corresponding to the existing `.ck-tooltip_sw`](https://github.com/ckeditor/ckeditor5/blob/i/8335/packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/tooltip/tooltip.css#L84-L98) and documented in the [`TooltipView#position` property docs](https://github.com/ckeditor/ckeditor5/blob/i/8335/packages/ckeditor5-ui/src/tooltip/tooltipview.js#L57).